components/footer.js

```diff
@@ -1,27 +1,21 @@
 import React from 'react'
 import glamorous from 'glamorous'
 import {Anchor} from '../components/styled-links'
-import * as colors from '../styles/colors'
 
 const Footer = glamorous.footer({
-  marginLeft: 'auto',
-  marginRight: 'auto',
-  marginTop: '5rem',
-  width: '100%',
-  maxWidth: 600,
   paddingTop: 10,
   paddingBottom: 10,
   fontSize: '0.6em',
   textAlign: 'center',
-  borderTop: `1px solid ${colors.faded}`,
+  background: 'white',
```

Comment:
For all colors we should pass in the theme and use the theme colors. If we ever change the shade of white (or whatever) we'll be glad we did it this way.

```js
const Footer = glamorous.footer((props, theme) => ({
   ...
   background: theme.colors.white
}))
```